document_print_control/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+
+> - [Sygel](https://www.sygel.es):
+>
+> > - Juan Alberto Raja Martinez\<<juan.raja@sygel.es>\>
+> > - Valentín Vinagre Urteaga\<<valentin.vinagre@sygel.es>\>
```

Comment:
You should check this format. It's not correct

<img width="1538" height="614" alt="Captura de pantalla 2025-07-14 a las 15 30 21" src="https://github.com/user-attachments/assets/4cbc8fa2-b584-4958-bc77-71a2321246c5" />